source/ftrack_connect_pipeline_3dsmax/utils/custom_commands.py

```diff
@@ -288,9 +293,9 @@ def getReferenceNode(assetLink):
 
 ### TIME OPERATIONS ###
 
+
 def get_time_range():
-    ''' Return the start and end frame of the current scene'''
+    '''Return the start and end frame of the current scene'''
     start = rt.animationRange.start
     end = rt.animationRange.end
     return (start, end)
-
```

Comment:
Please remove this changes from this PRs, will collide later on when merging to story.